### ISSUE TITLE
Repair Android tests

### ISSUE THREAD

**Issue by henri-tremblay** [easymock/objenesis#78]
It might require to migrate to Android

**Comment by gitryder** [comment]
Can I try to do this?

**Comment by henri-tremblay** [comment]
@gitryder Of Course! Go for it! Normally if you launch with the -Pall profile, you will things falling over.

**Comment by gitryder** [comment]
I'm sorry, I'm new to contributing. I don't understand what you're saying. 

What's -Pall profile?

**Comment by henri-tremblay** [comment]
when launching maven build: `mvn verify -Pall`. Otherwise the android module isn't compiled. The `README.md` should explain how to work with Android.

**Comment by NyorJa** [comment]
Can I check this also? Thanks.

**Comment by henri-tremblay** [comment]
All help is appreciated. Start by getting a working build. I can give move details afterwards. Some working example could be `https://github.com/raphw/byte-buddy/tree/master/byte-buddy-android-test`

**Comment by yogurtearl** [comment]
The gradle to maven migration fails with an NPE on this project. :/ 
File here: https://github.com/gradle/gradle/issues/14974

**Comment by yogurtearl** [comment]
Thoughts on just moving `tck-android` to gradle as a first step? 
would probably have to run it with `mvn install -Pall && ./gradlew connectedAndroidTest`

**Comment by yogurtearl** [comment]
Here is a basic gradle setup that let's you run the Android test. -> https://github.com/easymock/objenesis/pull/82

**Comment by nach-o-man** [comment]
@henri-tremblay I couldn't see any failing tests. Is this issue up-to-date?
Also created #84 with instructions for windows user on how-to run android tck
